settings.ts

```diff
@@ -46,7 +48,16 @@ export class NLSyntaxHighlightSettingTab extends PluginSettingTab {
 
 		containerEl.empty();
 
-
+		new Setting(containerEl)
+		.setName("Active")
```

Comment:
Can we change this to "Enabled" instead? I feel it's more widely used and understood than "Active".

Reply:
Agreed, changed in newest commit